docs/src/extension/write.md

```diff
@@ -1,3 +1,33 @@
 # Writing Extensions with PGRX
 
 <!-- TODO: write all of this -->
```

Comment:
This TODO looks done already, right? Or, at least, it'd become ``<!-- TODO: write more of this -->``